src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -333,6 +342,10 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     const activePlugins = bestOfPlugins(accountPlugins, accountReferral, undefined)
     plugins = plugins.filter(plugin => !activePlugins.disabled[plugin.pluginId])
 
+    if (!this.state.hasWyreAccountHack) {
+      plugins = plugins.filter(plugin => plugin.pluginId !== 'wyre')
+    }
+
```

Comment:
I see. So the scene launches without Wyre, and once the check passes (async), we update the list to include it. This seems right to me.

Reply:
Yep!